lib/src/main/java/uk/gov/di/ipv/cri/passport/library/service/ConfigurationService.java

```diff
@@ -1,5 +1,7 @@
 package uk.gov.di.ipv.cri.passport.library.service;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
These 2 imports aren't being used and can be removed

Reply:
Removed